src/firmware_new/boards/frankie_v1/usart.c

```diff
@@ -0,0 +1,117 @@
+/**
+ ******************************************************************************
+ * File Name          : USART.c
+ * Description        : This file provides code for the configuration
+ *                      of the USART instances.
+ ******************************************************************************
+ * @attention
+ *
+ * <h2><center>&copy; Copyright (c) 2020 STMicroelectronics.
+ * All rights reserved.</center></h2>
+ *
+ * This software component is licensed by ST under Ultimate Liberty license
+ * SLA0044, the "License"; You may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at:
+ *                             www.st.com/SLA0044
```

Comment:
Nit: Why's the URL got such weird whitespacing?

Reply:
autogenerated license :( its a cube problem, if I change it it will bounce right back the next time someone regenerates